library/src/yaml.rs

```diff
@@ -12,6 +12,8 @@ pub struct YamlConfig {
     pub base_url: Option<String>,
     /// Update behavior. Defaults to true if not set.
     pub auto_update: Option<bool>,
+    /// Path for the public key to verify the patch.
```

Comment:
```suggestion
    /// Path to a public key that will be used to verify patch hash signatures.
```

Reply:
Also, what format are we expecting this to be in?

Reply:
This is just a string path of the public key file for now. Did you had something else in mind?

Reply:
Right, what should that file contain? Raw bytes? A base64-encoded string?

Reply:
Ah yes, for my testing I am using that .pem file. Which I believe is just a text file?

I am not sure though what would be the best case though?

Reply:
Yep, just document it in the code :)